source/_posts/how-to-automate-deployments-of-ruby-application-updates-using-capistrano.md

```diff
@@ -0,0 +1,211 @@
+---
+title: How to automate deployments of Ruby Application updates using Capistrano
```

Comment:
Title   tag character should be 50-60 character, because more than 60 character not countable by the google